### PR TITLE
Make a PR template

### PR DESCRIPTION
**Pull request by WillEngler**
Look at this unstructured free text. Am I forgetting important information? What if we had a lightweight PR template that gave a little helpful nudge to PR writers but wasn't too opinionated? That would really be something.

----
:books: Documentation preview :books:: https://garden-ai--76.org.readthedocs.build/en/76/

### CODE REVIEW

**Review by ascourtas — COMMENTED** [review]
This looks great! Maybe also add a specific callout to link to any related issues, PRs, or blockers.

**Comment by WillEngler** [comment]
Thanks @ascourtas, added that

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+A quick summary of what this PR is meant to do.
+
+## Discussion
+
+More context that you think is important for reviewers or our future selves looking at this PR.
+
+## Testing
+
+How did you test this? If a reviewer pulled this branch down and wanted to kick the tires, how would they do so?
```
Comment (Aadit-Ambadkar) [inline .github/PULL_REQUEST_TEMPLATE.md:13]:
```suggestion
How did you test this? If a reviewer wanted to test this, what steps should they take to do so?
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Link to any related issues, companion PRs, or blockers. eg, `Fixes #37`
+
+## Overview
+
+A quick summary of what this PR is meant to do.
+
+## Discussion
+
+More context that you think is important for reviewers or our future selves looking at this PR.
```
Comment (Aadit-Ambadkar) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
Any additional context that you think is important for reviewers or future users/contributors looking at this PR.
```

**Review by Aadit-Ambadkar — COMMENTED** [review]
Make the language more clear

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (WillEngler) [reply .github/PULL_REQUEST_TEMPLATE.md:13]: Thanks for the suggestions :) I pulled in the other suggestion but on this one I think the original better conveys my intent.

**Comment by OwenPriceSkelly** [comment]
lgtm!

**Review by OwenPriceSkelly — APPROVED** [review]

**Pull request merged**